src/components/Widgets/WidgetSkeleton.style.ts

```diff
@@ -6,6 +6,7 @@ export const WidgetSkeletonContainer = styled(WidgetContainer)(({ theme }) => ({
   display: 'flex',
   flexDirection: 'column',
   alignItems: 'center',
+  maxWidth: 416,
```

Comment:
Would love to avoid those hardcoded values in the future and work with grid/flex instead

Reply:
I agree, but I cannot think of a proper way to avoid it right now. I´d love to hear your inputs, though to improve it.

Here, I built a Skeleton to represent something similar as the widget, which has a default width of 416px. I double-checked on the playground https://playground.li.fi/